### PR TITLE
Add distribution maintainers to maintainers files

### PR DESCRIPTION
**Pull request by dmcgowan**
Added to root maintainers file as well as individual directories with distribution code

### CODE REVIEW

**Comment by dmp42** [comment]
cc @kencochrane following our discussion

File: MAINTAINERS [hunk 1]
```diff
@@ -340,6 +340,9 @@ made through a pull request.
 				"vbatts",
 				"joffrey",
 				"samalba"
+				"sday"
```
Comment (jfrazelle) [inline MAINTAINERS]: needs commas

**Comment by dmcgowan** [comment]
@jfrazelle fixed

**Comment by jfrazelle** [comment]
LGTM ping @crosbymichael

**Comment by tiborvass** [comment]
MAINAINTERS Ping @shykes

**Comment by icecrime** [comment]
LGTM

**Comment by icecrime** [comment]
Validated by @crosbymichael (sitting next to me).

**Pull request merged**